sound/soc/sof/ipc4-topology.c

```diff
@@ -943,6 +944,21 @@ static int sof_ipc4_widget_setup_comp_process(struct snd_sof_widget *swidget)
 
 	sof_ipc4_widget_update_kcontrol_module_id(swidget);
 
+	pipe_widget = swidget->spipe->pipe_widget;
+
+	/*
+	 * A Data Processing (DP) domain widget can be scheduled to run on a secondary core while
+	 * it's pipeline runs the primary core. In this case, the secondary core could get powered
```

Comment:
the pipeline has nothing to do with cores. Never has, never will.

I think you meant "a data processing domain widget can be scheduled to run on a secondary core while other widgets of the same pipeline run on the primary core" ?

It would be useful to also explain why LL is not impacted, or what's so special about DP?

Reply:
Special is - 
a pipeline has a core it is bind to, All LL modules that belongs to the pipeline are bind to the same core as the pipeline itself.
One of DP principals was that a single module from the pipeline can be offloaded to secondary core and executed there, i.e.

LL1 (core0) ---> LL2(core0) ---> DP1 (core1) ---> LL3 (core0) --> DP2 (core2) --> LL4 (core0)